ui/orbital-stats.cpp

```diff
@@ -70,10 +72,54 @@ static const orbital_stats_device_info_t devices[] = {
     // Liverpool
     { UI_DEVICE_LIVERPOOL_GC,   "1002:9920", "Liverpool GC" },
     { UI_DEVICE_LIVERPOOL_HDAC, "1002:9921", "Liverpool HDAC" },
-};
+}};
```

Comment:
Is there any reason for duplicating the braces here?

Reply:
Yeah, it's that's how you're supposed to do it because of the nested brace initialization.
You get this otherwise:
```
ui/orbital-stats.cpp:75:1: error: too many initializers for 'const std::array<orbital_stats_device_info_t, 10>'
 };
 ^
```